pinot-broker/src/main/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManager.java

```diff
@@ -340,7 +435,7 @@ private boolean tryAcquireToken(String tableNameWithType, QueryQuotaEntity query
     // Emit the qps capacity utilization rate.
     int numHits = queryQuotaEntity.getQpsTracker().getHitCount();
     if (!rateLimiter.tryAcquire()) {
-      LOGGER.info("Quota is exceeded for table: {}. Per-broker rate: {}. Current qps: {}", tableNameWithType,
+      LOGGER.info("Quota is exceeded for table/database: {}. Per-broker rate: {}. Current qps: {}", resourceName,
```

Comment:
I think it probably makes more sense to separate the log lines for table and database query quota limits to make it more clear to users whether the table or the database query quota has been exceeded?

Reply:
Ah never mind, just noticed that we do that in the broker request handler anyway, so this isn't too important here.